### PR TITLE
i18n(es): Translated `from-pelican.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `from-pelican.mdx` to spanish 🚀

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-pelican.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+[Pelican](https://getpelican.com) es un generador de sitios estáticos de código abierto construido en Python.
+
+## Principales similitudes entre Pelican y Astro
+
+Pelican y Astro comparten algunas similitudes que te ayudarán a migrar tu proyecto:
+
+- Pelican y Astro son generadores de sitios estáticos, idealmente adecuados para [sitios web centrados en contenido](/es/concepts/why-astro/#enfocado-en-el-contenido) como blogs.
+
+- Pelican y Astro tienen soport incorporado para [escribir Markdown](/es/guides/content/#autoría-mdx), incluyendo propiedades frontmatter YAML para metadatos de página. Sin embargo, Astro tiene muy pocas propiedades frontmatter reservadas en comparación con Pelican. Aunque muchas de tus propiedades frontmatter existentes en Pelican no serán "especiales" en Astro, puedes continuar usando tus archivos Markdown existentes y valores frontmatter.
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-pelican.mdx:20]:
```suggestion
- Pelican y Astro tienen soporte incorporado para [escribir Markdown](/es/guides/content/#autoría-mdx), incluyendo propiedades frontmatter YAML para metadatos de página. Sin embargo, Astro tiene muy pocas propiedades frontmatter reservadas en comparación con Pelican. Aunque muchas de tus propiedades frontmatter existentes en Pelican no serán "especiales" en Astro, puedes continuar usando tus archivos Markdown existentes y valores frontmatter.
```

File: src/content/docs/es/guides/migrate-to-astro/from-pelican.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Principales diferencias entre Pelican y Astro
+
+Cuando reconstruyas tu sitio Pelican a Astro, notarás algunas diferencias importantes:
+
+- Pelican admite escribir contenido en Markdown y reStructured Text (`.rst`). Astro admite [crear páginas a partir de archivos Markdown y MDX](/es/guides/content/), pero no admite reStructured Text.
+
+- Pelican usa archivos HTML y sintaxis Jinja para la plantilla. [Astro syntax](/es/core-concepts/astro-components/) es un subconjunto de HTML similar a JSX. Todo el HTML válido es una sintaxis válida para `.astro`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/migrate-to-astro/from-pelican.mdx:28]:
```suggestion
- Pelican usa archivos HTML y sintaxis Jinja para la plantilla. [La sintaxis de Astro](/es/core-concepts/astro-components/) es un subconjunto de HTML similar a JSX. Todo el HTML válido es una sintaxis válida para `.astro`.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**